graphql/schema/types/distro.graphql

```diff
@@ -42,6 +42,31 @@ enum DispatcherVersion {
   REVISED_WITH_DEPENDENCIES
 }
 
+enum Arch {
```

Comment:
Also noticing that a few of these are no longer in use! Here's what's being used:
```
> db.distro.distinct("arch")
> [
  'darwin_amd64',
  'darwin_arm64',
  'linux_amd64',
  'linux_arm64',
  'linux_ppc64le',
  'linux_s390x',
  'windows_amd64'
]
```
So neither 32-bit architecture is in use (Windows or Linux).

Brian McCarthy is ooo this week but I'm going to follow up when he's back to see if we can remove these. If you'd like to omit them in this PR that's probably okay, and we can add them in if necessary.